pages/about/programs/_slug.vue

```diff
@@ -1,33 +1,34 @@
 <template>
-    <section class="page-program-detail">
+    <main class="page-program-detail">
         <nav-breadcrumb
             to="/about/program"
             :title="page.title"
             parent-title="Programs"
         />
 
-        <banner-text
-            v-if="!page.heroImage || page.heroImage.length == 0"
-            class="banner-text"
-            :title="page.title"
-            :text="page.text"
-            :button-text="parsedButtonText"
-            :to="parsedButtonTo"
-            :email="page.email"
-            :phone="page.phoneNumber"
-            :staff-directory-link="parsedStaffDirectory"
-        />
+        <section-wrapper class="section-banner">
+            <banner-text
+                v-if="!page.heroImage || page.heroImage.length == 0"
+                :title="page.title"
+                :text="page.text"
+                :email="page.email"
+                :phone="page.phoneNumber"
+                :staff-directory-link="parsedStaffDirectory"
+                :button-text="parsedButtonText"
+                :to="parsedButtonTo"
+            />
+        </section-wrapper>
 
         <banner-header
```

Comment:
swap banner-header with banner-text in the section-wrapper

Reply:
```html
       <banner-text
            v-if="!page.heroImage || page.heroImage.length == 0"
            :title="page.title"
            :text="page.text"
            :email="page.email"
            :phone="page.phoneNumber"
            :staff-directory-link="parsedStaffDirectory"
            :button-text="parsedButtonText"
            :to="parsedButtonTo"
        />

        <section-wrapper class="section-banner">
            <banner-header
                v-if="page.heroImage && page.heroImage.length == 1"
                :image="page.heroImage[0].image[0]"
                :title="page.title"
                :text="page.text"
                :email="page.email"
                :phone="page.phoneNumber"
                :staff-directory-link="parsedStaffDirectory"
                :prompt="parsedButtonText"
                :to="parsedButtonTo"
                :align-right="true"
            />
        </section-wrapper>
```